dagshub/data_engine/client/gql_queries.py

```diff
@@ -41,7 +41,7 @@ def datasource_params(id: Optional[Union[int, str]], name: Optional[str]) -> Dic
 
     @staticmethod
     @functools.lru_cache()
-    def datasource_query(include_metadata: bool) -> str:
+    def datasource_query(include_metadata: bool) -> GqlQuery:
```

Comment:
If you're changing the return type on this query, change it on every other query/mutation too.